src/redux/actions/auth.js

```diff
@@ -0,0 +1,56 @@
+import ACTIONS from '../actionTypes'
+
+const signIn = credentials => {
+  return (dispatch, getState, { getFirebase }) => {
+    const firebase = getFirebase()
+    firebase
+      .auth()
+      .signInWithEmailAndPassword(credentials.email, credentials.password)
+      .then(() => {
+        dispatch({ type: ACTIONS.LOGIN_SUCCESS })
```

Comment:
I'm assuming current user info will be accessible from firestore after this?

Reply:
Yes
<img width="917" alt="screenshot 2019-01-08 23 36 04" src="https://user-images.githubusercontent.com/1489583/50863209-3ebf5d80-139e-11e9-9de0-d2cc9c25ead2.png">